src/keys/KeyManager.ts

```diff
@@ -120,11 +113,10 @@ class KeyManager {
     this.rootCertPath = path.join(keysPath, 'root.crt');
     this.rootCertsPath = path.join(keysPath, 'root_certs');
     this.dbKeyPath = path.join(keysPath, 'db.key');
-    this.vaultKeyPath = path.join(keysPath, 'vault.key');
+    this.litmusMessagePath = path.join(keysPath, 'litmusMessage');
```

Comment:
Is this being used atm? I think it was forgotten to be implemented...? If it was done elsewhere, it should be done here, not in vaults.

Reply:
I remember talking about this with @scottmmorris before. It was intended to be `canaryPath`. It just seems it was never fully implemented. If so, then you can just remove it, and we can add it in later feature.